src/main/java/com/servicewizard/model/ServiceModel.java

```diff
@@ -0,0 +1,37 @@
+package com.servicewizard.model;
+
+import java.util.LinkedList;
+import java.util.List;
+
+public class ServiceModel {
+
+	public String getApiName() {
+		return apiName;
+	}
+
+	public void setApiName(String apiName) {
+		this.apiName = apiName;
+	}
+
+	public String getIntroText() {
+		return introText;
+	}
+
+	public void setIntroText(String introText) {
+		this.introText = introText;
+	}
+
+	public List<Service> getServices() {
+		return services;
+	}
+
+	private String apiName;
+
+	private String introText;
+
+	private final List<Service> services;
+
+	public ServiceModel() {
```

Comment:
Can we move constructors to above the private members, but at below the public methods?